test failures on mips64 in dnrm2_tiny and dnrm2_inf

In OpenBLAS 0.3.21, on mips64, the Debian package fails to build because there are test failures in the utests for `dnrm2_tiny` and `dnrm2_inf`:
```
TEST 32/37 dnrm2:dnrm2_tiny [FAIL]
  ERR: test_dnrm2.c:65  expected 0.000e+00, got inf (diff -inf, tol 1.000e-13)
TEST 33/37 dnrm2:dnrm2_inf [FAIL]
  ERR: test_dnrm2.c:52  expected inf, got 0.000e+00 (diff inf, tol 1.000e-13)
```
(see https://buildd.debian.org/status/fetch.php?pkg=openblas&arch=mips64el&ver=0.3.21%2Bds-1&stamp=1662336716&raw=0 for the full build log).

I noticed that commit cce4b1d9562d60b875982ce9587c68fbac666ec8 by @XiWeiGu was supposed to fix `dnrm2_tiny`.

Interestingly, if I revert that very commit, then it fixes `dnrm2_inf`:
```
TEST 32/37 dnrm2:dnrm2_tiny [FAIL]
  ERR: test_dnrm2.c:65  expected 0.000e+00, got inf (diff -inf, tol 1.000e-13)                                                    
TEST 33/37 dnrm2:dnrm2_inf [OK]
```

So it seems that the logic of commit cce4b1d9562d60b875982ce9587c68fbac666ec8 is wrong, and that instead of fixing `dnrm2_tiny`, it broke `dnrm2_inf`.

Compiling the latest code on the 3A4000 with Loongnix GNU/Linux 20 RC3 presents the following problems：

> /usr/include/mips64el-linux-gnuabi64/gnu/stubs.h:41:11: 致命错误：gnu/stubs-n64_hard_2008.h：没有那个文件或目录
>  #nclude <gnu/stubs-n64_hard_2008.h>
>            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
> In file included from /usr/include/features.h:448,
>                  from /usr/include/mips64el-linux-gnuabi64/bits/libc-header-start.h:33,
>                  from /usr/include/stdio.h:27,
>                  from axpy.c:39:
> /usr/include/mips64el-linux-gnuabi64/gnu/stubs.h:41:11: 致命错误：gnu/stubs-n64_hard_2008.h：没有那个文件或目录
>  #include <gnu/stubs-n64_hard_2008.h>
>            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
> 编译中断。
> 编译中断。
> In file included from /usr/include/features.h:448,
>                  from /usr/include/mips64el-linux-gnuabi64/bits/libc-header-start.h:33,
>                  from /usr/include/stdio.h:27,
>                  from copy.c:39:
> /usr/include/mips64el-linux-gnuabi64/gnu/stubs.h:41:11: 致命错误：gnu/stubs-n64_hard_2008.h：没有那个文件或目录
>  #include <gnu/stubs-n64_hard_2008.h>
>            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
> 编译中断。
> make[1]: *** [Makefile:822：saxpy.o] 错误 1
> make[1]: *** 正在等待未完成的任务....
> make[1]: *** [Makefile:849：sscal.o] 错误 1
> make[1]: *** [Makefile:876：scopy.o] 错误 1
> In file included from /usr/include/features.h:448,
>                  from /usr/include/mips64el-linux-gnuabi64/bits/libc-header-start.h:33,
>                  from /usr/include/stdio.h:27,
>                  from swap.c:39:
> /usr/include/mips64el-linux-gnuabi64/gnu/stubs.h:41:11: 致命错误：gnu/stubs-n64_hard_2008.h：没有那个文件或目录
>  #include <gnu/stubs-n64_hard_2008.h>
>            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
> 编译中断。
> make[1]: *** [Makefile:894：sswap.o] 错误 1
> 
So I reset to [cc4b1d](https://github.com/xianyi/OpenBLAS/commit/cce4b1d9562d60b875982ce9587c68fbac666ec8) temporarily, all the utest passed:

> TEST 1/35 max:smax_zero [OK]
> TEST 2/35 max:dmax_positive [OK]
> TEST 3/35 max:smax_negative [OK]
> TEST 4/35 min:smin_zero [OK]
> TEST 5/35 min:dmin_positive [OK]
> TEST 6/35 min:smin_negative [OK]
> TEST 7/35 amax:damax [OK]
> TEST 8/35 amax:samax [OK]
> TEST 9/35 ismax:negative_step_2 [OK]
> TEST 10/35 ismax:positive_step_2 [OK]
> TEST 11/35 ismin:negative_step_2 [OK]
> TEST 12/35 ismin:positive_step_2 [OK]
> TEST 13/35 drotmg:drotmg_D1_big_D2_big_flag_zero [OK]
> TEST 14/35 drotmg:rotmg_D1eqD2_X1eqX2 [OK]
> TEST 15/35 drotmg:rotmg_issue1452 [OK]
> TEST 16/35 drotmg:rotmg [OK]
> TEST 17/35 axpy:caxpy_inc_0 [OK]
> TEST 18/35 axpy:saxpy_inc_0 [OK]
> TEST 19/35 axpy:zaxpy_inc_0 [OK]
> TEST 20/35 axpy:daxpy_inc_0 [OK]
> TEST 21/35 zdotu:zdotu_offset_1 [OK]
> TEST 22/35 zdotu:zdotu_n_1 [OK]
> TEST 23/35 dsdot:dsdot_n_1 [OK]
> TEST 24/35 swap:cswap_inc_0 [OK]
> TEST 25/35 swap:sswap_inc_0 [OK]
> TEST 26/35 swap:zswap_inc_0 [OK]
> TEST 27/35 swap:dswap_inc_0 [OK]
> TEST 28/35 rot:csrot_inc_0 [OK]
> TEST 29/35 rot:srot_inc_0 [OK]
> TEST 30/35 rot:zdrot_inc_0 [OK]
> TEST 31/35 rot:drot_inc_0 [OK]
> TEST 32/35 dnrm2:dnrm2_tiny [OK]
> TEST 33/35 dnrm2:dnrm2_inf [OK]
> TEST 34/35 fork:safety [OK]
> TEST 35/35 fork:safety_after_fork_in_parent [OK]
> RESULTS: 35 tests (35 ok, 0 failed, 0 skipped) ran in 530 ms

--- a/kernel/mips64/dnrm2.S
+++ b/kernel/mips64/dnrm2.S
@@ -90,7 +90,7 @@
 	//Init INF
 	lui     TEMP, 0x7FF0
 	dsll    TEMP, TEMP, 32
-	MTC1    TEMP, INF
+	MTC     TEMP, INF
 
 	LD	a1,  0 * SIZE(X)
 	daddiu	N, N, -1

